tests/scanner/scanner_builder_test.py

```diff
@@ -88,8 +88,29 @@ def testTwoEnabled(self, mock_bucket_rules_engine, mock_iam_rules_engine):
             FAKE_GLOBAL_CONFIGS, fake_runnable_scanners.TWO_ENABLED,
             mock.MagicMock(), '', FAKE_TIMESTAMP)
         runnable_pipelines = builder.build()
-
+        
         self.assertEquals(2, len(runnable_pipelines))
         expected_pipelines = ['BucketsAclScanner', 'IamPolicyScanner']
         for pipeline in runnable_pipelines:
             self.assertTrue(type(pipeline).__name__ in expected_pipelines)
+
+    @mock.patch('google.cloud.forseti.scanner.scanner_builder.LOGGER',
+                autospec=True)
+    def testNonExistEnabled(self, mock_logging):
```

Comment:
The test method name here would be more clear as:
`testNonExistentScannerIsHandled`

As in the other test above, in addition to the logging being called, it would also be good to assert on the number of scanners in the output of builder.build(), and what exact scanners are returned.
1: assert the count of scanners is expected
2: do a loop and assert the outputted scanners are found in the test data map
See above test cases for example.
```
        self.assertEquals(2, len(runnable_pipelines))
        expected_pipelines = ['BucketsAclScanner', 'IamPolicyScanner']
        for pipeline in runnable_pipelines:
              self.assertTrue(type(pipeline).__name__ in expected_pipelines)
```

Reply:
modified as suggested